### DISCUSSION TITLE
lz4 and tar.lz4 extraction support

### DISCUSSION THREAD

**Discussion by spiderman666** [aonez/Keka#1442] [Q&A]
RE: issue [#85](https://github.com/aonez/Keka/issues/85) Extraction support for .lz4 and tar.lz4 archives please? Many thanks!

**Comment by aonez** [comment]
Not sure why it take so long:

https://github.com/aonez/Keka/releases/tag/dev-test-builds
https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-v1.4.0-dev.r5400.7z

**Reply by spiderman666** [reply]
@aonez Excellent, thank you very much!
